Capri.Database/Entities/Promoter.cs

```diff
@@ -12,6 +12,12 @@ public class Promoter : IEntity
     {
         [Key]
         public Guid Id { get; set; }
+        [Required]
+        public string FirstName { get; set; }
+        [Required]
+        public string LastName { get; set; }
```

Comment:
Aren't `FirstName` and `LastName` in `ApplicationUser`

Reply:
No, they aren't